web/modules/custom/nlc_prototype/nlc_prototype.module

```diff
@@ -57,3 +58,23 @@ function nlc_prototype_page_attachments(array &$page) {
     $page['#attached']['library'][] = 'nlc_prototype/google_analytics';
   }
 }
+
+/**
+ * Implements hook_search_api_query_alter().
+ * https://www.drupal.org/forum/support/post-installation/2016-06-09/solved-how-do-i-make-a-custom-module-trigger-a-function
+ */
+function facets_summary_search_api_query_alter(QueryInterface &$query) {
```

Comment:
This function is uses `hook_search_api_query_alter()` and is called `facets_summary_search_api_query_alter()` but it's in the `nlc_prototype.module` file so shouldn't it be called `nlc_prototype_search_api_query_alter()` instead? There may be a future function name clash with `facets_summary` module otherwise.